ome/provider.go

```diff
@@ -219,6 +219,7 @@ func (p *omeProvider) DataSources(ctx context.Context) []func() datasource.DataS
 		NewVlanNetworkDataSource,
```

Comment:
---

### Code Review Report

The provided code diff is relatively small, but there are some suggestions for improvement.

#### Naming Conventions

* The added line `NewAppCertDataSource` could benefit from following the standard naming convention for Go variables and functions, which is to use camelCase. Instead of `NewAppCertDataSource`, consider renaming it to `newAppCertDataSource`.

#### Code Organization

* The `DataSources` function seems to be a collection of data sources. Consider using a struct to hold these data sources and then return an instance of that struct. This would make the code more organized and easier to manage.

Here's an example of how the code could be refactored:
```go
type OmeProviderDataSources struct {
    NewVlanNetworkDataSource func() datasource.DataSource
    NewConfigurationReportDataSource func() datasource.DataSource
    NewDeviceDatasource func() datasource.DataSource
    NewAppCertDataSource func() datasource.DataSource
}

func (p *omeProvider) DataSources(ctx context.Context) []func() datasource.DataSource {
    return []func() datasource.DataSource{
        p.newVlanNetworkDataSource,
        p.newConfigurationReportDataSource,
        p.newDeviceDatasource,
        p.newAppCertDataSource,
    }
}
```

### Code Snippet:
```go
@@ -219,6 +219,7 @@ func (p *omeProvider) DataSources(ctx context.Context) []func() datasource.DataS
 		NewVlanNetworkDataSource,
 		NewConfigurationReportDataSource,
 		NewDeviceDatasource,
+		return []func() datasource.DataSource{
+			p.newAppCertDataSource,
+		}
 	}
 }
```

By following these suggestions, the code will become more readable and maintainable.